src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_scheduleCourtHearing.java

```diff
@@ -0,0 +1,269 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.ErrorResponse;
+import net.syscon.elite.executablespecification.steps.AuthTokenHelper;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.CREATED;
+
+public class OffenderMovementsResourceImplIntTest_scheduleCourtHearing extends ResourceTest {
+
+    @Test
+    public void schedules_court_hearing() {
+        final var token = authTokenHelper.getToken(AuthTokenHelper.AuthToken.NORMAL_USER);
+
+        final var request = createHttpEntity(token, Map.of(
+                "fromPrisonLocation", "LEI",
+                "toCourtLocation", "COURT1",
+                "courtHearingDateTime", "2030-03-11T14:00:00.000Z",
+                "comments", "some comments"
+        ));
+
+        final var response = testRestTemplate.exchange(
+                "/api/bookings/-1/prison-to-court-hearings",
+                HttpMethod.POST,
+                request,
+                new ParameterizedTypeReference<String>() {
+                });
+
+
+        assertThat(response.getStatusCode()).isEqualTo(CREATED);
+        assertThat(getBodyAsJsonContent(response)).extractingJsonPathNumberValue("$.id").isNotNull();
```

Comment:
This value is populated via a sequence on the DB so cannot guarantee the version.